de1plus/de1_comms.tcl

```diff
@@ -138,97 +140,109 @@ proc append_to_de1_list {address name type} {
 	msg "Scan found DE1: $address"
 	set ::de1_device_list $newlist
 	catch {
-		fill_ble_listbox
+		fill_de1_listbox
 	}
 }
 
-proc append_to_de1_list {address name type} {
```

Comment:
How did we manage to end up with the same function multiple times?
Maybe patch files are rightfully not the future :D

Reply:
It can be a user error, simply copy/paste. I've made this error in the past many times.

Tcl allows you to redefine a proc{} at any time, without a warning.